# clear soft limits (no parameters required)
---
# response status code
int32 status_code
